### ISSUE TITLE
What is Frame Length x Sample Rate?

### ISSUE THREAD

**Issue by dilky97** [facebookresearch/pytorchvideo#134]
Can someone explain what is Frame Length x Sample Rate in SlowFast Networks? 
So far what I have understood is in the Slow Only Version the two pathways should have a same frame rate.

**Comment by kalyanvasudev** [comment]
Yes, the input to both networks should be of same shape. Here is some quick info,
Frame Length = Number of frames in the video clip.
Sample Rate = Stride length in the original video at which the frames are sampled for the clip.

For more details please refer to the original SlowFast paper - https://arxiv.org/pdf/1812.03982.pdf
